iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -126,15 +126,6 @@ data_replication_factor=1
 # Datatype: Double
 # data_region_per_data_node=5.0
 
-
-# The policy of cluster RegionGroups' leader distribution.
-# E.g. we should balance cluster RegionGroups' leader distribution when some DataNodes are shutdown or re-connected.
-# These policies are currently supported:
-# 1. GREEDY(Distribute leader through a simple greedy algorithm, might cause unbalance)
-# 2. MIN_COST_FLOW(Default, distribute leader through min cost flow algorithm)
-# Datatype: String
-# leader_distribution_policy=MIN_COST_FLOW
```

Comment:
why remove this one?

Reply:
It seems that the GREEDY algorithm will never be used